tests/sentry/ratelimits/utils/test_get_ratelimit_key.py

```diff
@@ -87,6 +88,31 @@ def test_organization(self):
             == f"org:default:OrganizationGroupIndexEndpoint:GET:{install.organization_id}"
         )
 
+    def test_api_token(self):
+        token = ApiToken.objects.create(user=self.user, scope_list=["event:read", "org:read"])
+        self.request.auth = token
+        self.request.user = self.user
+        assert (
+            get_rate_limit_key(
+                self.view, self.request, self.rate_limit_group, self.rate_limit_config
+            )
+            == f"user:default:OrganizationGroupIndexEndpoint:GET:{self.user.id}"
+        )
+
+    def test_authenticated_token(self):
+        # Ensure AuthenticatedToken kinds are registered
+        import sentry.services.hybrid_cloud.auth.impl  # noqa: F401
```

Comment:
Specifically https://github.com/getsentry/sentry/blob/master/src/sentry/services/hybrid_cloud/auth/impl.py#L367-L370

This feels weird but I couldn't immediately come up with a better approach